tests/check_flatpak_extra_languages.pm

```diff
@@ -0,0 +1,32 @@
+# vi: set shiftwidth=4 tabstop=4 expandtab:
+use base 'installedtest';
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    my $self = shift;
+    my $default_extra_languages = "ar;bn;en;es;fr;id;pt;th;vi;zh";
+    my $extra_languages = get_var('EOS_IMAGE_FLATPAK_LOCALES', '');
+    $extra_languages =~ s/\ /;/ig;
+
+    # Test extra-languages key is present
+    $self->user_console();
+    if ($extra_languages eq $default_extra_languages) {
+        assert_script_run('flatpak config --system --get extra-languages', 60, '*unset*');
+    } elsif (!$extra_languages) {
+        assert_script_run('flatpak config --system --get extra-languages', 60, $extra_languages);
+    }
+
+    $self->exit_user_console();
+}
+
+sub test_flags {
+    # 'fatal'          - abort whole test suite if this fails (and set overall state 'failed')
+    # 'ignore_failure' - if this module fails, it will not affect the overall result at all
+    # 'milestone'      - after this test succeeds, update 'lastgood'
+    # 'norollback'     - don't roll back to 'lastgood' snapshot if this fails
+    return { ignore_failure => 1 };
```

Comment:
Why is this `ignore_failure => 1` rather than `fatal => 1` like almost all the other tests? This test is not expected to fail, so I don’t think `ignore_failure` is appropriate.

Reply:
To be honest I just copied that from another test and didn't have much thought of it :( Will fix it.